server/src/omtLinkProvider.ts

```diff
@@ -0,0 +1,173 @@
+import { dirname, isAbsolute, resolve } from 'path';
+import { DocumentLink, Position, Range } from 'vscode-languageserver';
+import { TextDocument } from 'vscode-languageserver-textdocument';
+import { readFileSync } from 'fs';
+import { glob } from 'glob';
+import { WorkspaceLookup } from './workspaceLookup';
+
+const omtUriMatch = /( +["']?)(.*\.omt)/;
+const declaredImportMatch = /( +)(module:)(.*):/;
+const importMatch = /^import:/g;
+const otherDeclareMatch = /^(\w+):/g;
+
+/**
+ * Provides DocumentLinks for the imports of an OMT file.
+ */
+export default class OMTLinkProvider {
+
+    constructor(private workspaceLookup: WorkspaceLookup) { }
+
+    provideDocumentLinks(document: TextDocument): DocumentLink[] {
```

Comment:
I see to types of private function declarations. with and without private

Reply:
I think you mean public right?
there are no private functions in omtLinkProvider (except for the non exported functions)

or did you mean in different classes? >> will check that